config/core/deployments/webhook-hpa.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: webhook
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
+spec:
+  minReplicas: 1
+  maxReplicas: 20
```

Comment:
Why 20? Should we even be setting anything above 1 in the stock yaml?

Reply:
Sorry that I'm new to this community and actually couldn't quite understand what you meant. What's a "stock yaml" and why we need a scaler if we only need at most 1 instance?

Reply:
Well we ship this file by default. Why did you decide on 20?

Reply:
I was trying to mimic how this was configured in activator-hpa.yaml.

Reply:
Yeah but they are not the same. I guess, it doesn't really matter, but I hope we don't need 20 instances of webhook. Probably 5 is good starting max number.

Reply:
Thanks for your suggestion, I just updated it.